### PR TITLE
Add -flags argument

### PR DESCRIPTION
**Pull request by ahmetb**
- `govvv -print-ldflags` just prints the value of -ldflags
  argument it passes to the go tool.
- Bump the version to 0.2.0

cc: @rakyll

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,11 @@ You can just pass a `-print` argument and `govvv` will just print the
 	    -ldflags \
 	    "-X main.GitCommit=57b9870 -X main.GitBranch=dry-run -X main.GitState=dirty -X main.Version=0.1.0 -X main.BuildDate=2016-08-08T20:50:21Z"
 
+Still don't want to wrap the `go` tool? Well, try `-print-ldflags` and run as:
+
+    $ go build -ldflags="$(govvv -print-ldflags)"
```
Comment (rakyll) [inline README.md]: Would be good to have an argument name less verbose. `govvv -flags`?

Comment (ahmetb) [inline README.md]: sounds good. Let me fix.

Comment (ahmetb) [inline README.md]: done

File: README.md [hunk 2]
```diff
@@ -53,6 +53,11 @@ You can just pass a `-print` argument and `govvv` will just print the
 	    -ldflags \
 	    "-X main.GitCommit=57b9870 -X main.GitBranch=dry-run -X main.GitState=dirty -X main.Version=0.1.0 -X main.BuildDate=2016-08-08T20:50:21Z"
 
+Still don't want to wrap the `go` tool? Well, try `-print-ldflags` and run as:
```
Comment (rakyll) [inline README.md:56]: Update the README too.

**Pull request merged**